index.html

```diff
@@ -182,8 +182,8 @@ <h4 class="px-3">
   </main>
 
   <!-- modal -->
-  <!--  -->
-  <div class="modal fade" id="book-form-modal" tabindex="-1" role="dialog">
+  <!-- class="modal fade" -->
```

Comment:
commented out class="modal fade"  from the div element so I could make sure that my results were pushing to the modal